### ISSUE TITLE
Textlabels colors dont work properly

### ISSUE THREAD

**Issue by S0P4** [altmp/altv-issues#1891]
### Description of the problem

As the title says, the color parameter for text labels its not working properly. The red and blue chanels are not working.

### Reproduction steps

```ts
alt.on('consoleCommand', (cmd: string, ...args: string[]) => {
	if (cmd === 'testtextlabel') {
		const r = args[0] ? parseInt(args[0]) : 255;
		const g = args[1] ? parseInt(args[1]) : 255;
		const b = args[2] ? parseInt(args[2]) : 255;
		const a = args[3] ? parseInt(args[3]) : 255;

		const color = new alt.RGBA(r, g, b, a);

		const txt = new alt.TextLabel('test', 'ChaletComprimeCologneEighty', 30, 1, alt.Player.local.pos, { x: 0, y: 0, z: 0 }, color, 1, color);
		txt.dimension = alt.Player.local.dimension;
	}
});

```

NOTE: The font has to be loaded.

Ex: 
* testtextlabel 255 0 0 (Gives a black textlabel)
* testtextlabel 0 255 0 (Gives a green textlabel)
* testtextlabel 0 0 255 (Gives a black textlabel)

### Expected behaviour

Self explanatory, the colors should be reflected properly in game.

### Operating system

Windows 11

### Version

dev/15.0-dev354

### Reproduction tested

- [X] I confirm that I have made sure that this issue is also present on the newest dev version

**Comment by xxshady** [comment]
Confirmed on 15.0-dev360 (note: you could use `alt.Utils.ConsoleCommand` for single console command)

**Comment by C0kkie** [comment]
Should be fixed in dev/15.0-dev479
